_config.yml

```diff
@@ -26,6 +26,7 @@ sass:
 plugins:
   - jekyll-seo-tag
   - jekyll-last-modified-at
+  - jekyll-sitemap
```

Comment:
the sitemap seems to work well, but is there a way to save it somewhere apart from _site? I'm hoping to use this sitemap file as part of a bigger sitemap index that im working on. it would be helpful to have this file accessible via a link that I can curl or something without having to have master pipeline clone and pull the repo every time it runs.